livox_ros_driver/package.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>livox_ros_driver</name>
-  <version>2.0.0</version>
+  <!-- 0.0.0 needed for debian packaging find/replace (see edge-cli) -->
+  <version>0.0.0</version>
```

Comment:
Since this is a fork, I'm wondering if this step is necessary / if edge-cli has the ability to "mutate" a pre-existing/ provided version number? 

If not, that's fine, but preserving the original versioning seems helpful to identify features/ easier version control